src/price_data.py

```diff
@@ -361,6 +363,18 @@ def _get_price_kraken(
 
                 if not data["error"]:
                     break
+                elif data["error"] == ['EGeneral:Invalid arguments']:
+                    # cancel if inverse pair has been tried
+                    if inverse:
+                        num_retries = 0
+                        continue
```

Comment:
This is hard to read. Either add an comment (explicitly state, that we want to use for for-else case) or raise the error directly. We can adjust the error message and say, that normal and inverse pair was tried.